client/src/components/OneGuide/index.js

```diff
@@ -1,15 +1,67 @@
-import React from 'react'
+import React, { Component, Fragment } from 'react'
```

Comment:
No need to destructure Fragment here as you're not doing it anywhere else.

Reply:
it's doesn't avaliabile